src/firecracker/swagger/firecracker.yaml

```diff
@@ -943,6 +967,17 @@ definitions:
       vsock:
         $ref: "#/definitions/Vsock"
 
+  Hotplug:
+    type: object
+    description:
+      Defines the configuration for hot plugging.
+    properties:
+      vcpu_count:
+        type: integer
+        minimum: 1
```

Comment:
what is the purpose of `minimum` here?

Reply:
Just identifies that `vcpu_count` can't be 0 or less.